manifests/t/TaskPlays/TSWModManager/2.2.1/TaskPlays.TSWModManager.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: TaskPlays.TSWModManager
+PackageVersion: 2.2.1
+InstallerLocale: en-US
+InstallerType: nullsoft
+Scope: user
+UpgradeBehavior: install
+ReleaseDate: 2022-09-01
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
`x86` validations are performed on a 32-bit OS.